src/scripts/update-firmware-binaries.js

```diff
@@ -234,20 +234,24 @@ const downloadAppBinaries = async (): Promise<*> => {
     repo: GITHUB_FIRMWARE_REPOSITORY,
   });
 
-  const versionText = new Buffer(versionResponse.content, 'base64').toString();
-  const startIndex = versionText.indexOf('| 0 ');
-  const endIndex = versionText.indexOf('\n\n', startIndex);
+  const versionText =
+    new Buffer(versionResponse.content, 'base64').toString() || '';
```

Comment:
why `|| ''`? if we don't have versionText, it means something bad happened during file downloading and we should throw an error.